.stylelintrc-styleguide.js

```diff
@@ -0,0 +1,298 @@
+  var config = {
+    "plugins": ["stylelint-prettier"],
+    "rules": {
+      "color-no-hex": true,
+      "declaration-property-value-blacklist": [
+        {
+          "/.*/": [
+            /rgba{0,1}\(/i,
+            /hsla{0,1}\(/i,
+            /hwb\(/i,
+            /gray\(/i,
+            // uses of web color names are not allowed (with 'white' and 'black' being the only exceptions)
+            /[^\$-\w]lavender/i,
+            /^lavender/i,
```

Comment:
to capture the instances when value of the CSS rule starts with a color name,

e.g.,

```css
property: colorname; // we don't have to worry about the whitespace there as stylelint seems to trim beginning and tailing whitespace before it runs the checks
```

Reply:
We don't need two rules per color here, `/[^\$-\w]?lavender/i,` captures both. However, rather than specifying this ourselves, could we just tap into https://github.com/stylelint/stylelint/tree/master/lib/rules/color-named instead and say `never`?

(since we have a $black and $white for use outside of _variables.scss)